templates/api-server.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "api-server",  
+  "description": "A LoopBack API server with local User auth",
+  "package": {
+    "version": "1.0.0",
+    "main": "server/server.js",
+    "scripts": {
+      "lint": "eslint .",
+      "start": "node .",
```

Comment:
> we should consider including "test": "mocha" in the default templates -- @bajtos thoughts?

@superkhau That would make sense to me only if we included some sample (smoke?) tests in the template too. If you think this is worth pursuing, then please open a new GH issue to discuss.